ginga/ImageView.py

```diff
@@ -23,6 +23,8 @@
 from ginga.canvas.types.layer import DrawingCanvas
 from ginga.util import io_rgb
 
+__all__ = ['ImageViewBase']
+
```

Comment:
AFAIK `__all__` only affects `from module import *`, which I think is what was tripping the `automodsum` directive that comes with the package template, because without this, it was attempting to find random things like `zip`, `map`, functions from another module that were imported, etc. As long as you don't do `... import *` in your own modules when you want to subclass etc (which is not recommended anyway), you should be okay.